app-editors/retext/retext-8.0.0.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please don't add pypy support before testing if it's actually supported. The
+# old compat matrix is no longer accessible as of 2021-02-13 but stated back
+# in 2020-07-05 that PyQt5 was explicitly not supported.
+PYTHON_COMPAT=( python3_{8,9,10} )
+
+inherit distutils-r1 optfeature qmake-utils virtualx xdg
+
+MY_PN="ReText"
+MY_P="${MY_PN}-${PV/_/~}"
+
+DESCRIPTION="Simple editor for Markdown and reStructuredText"
+HOMEPAGE="https://github.com/retext-project/retext https://github.com/retext-project/retext/wiki"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/retext-project/retext.git"
+else
+	SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+	S="${WORKDIR}/${MY_P}"
+
+	KEYWORDS="~amd64 ~riscv ~x86"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/chardet[${PYTHON_USEDEP}]
+	dev-python/docutils[${PYTHON_USEDEP}]
+	dev-python/markdown[${PYTHON_USEDEP}]
+	>=dev-python/markups-3.1.1[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/python-markdown-math[${PYTHON_USEDEP}]
+	dev-python/PyQt6[dbus,gui,printsupport,widgets,${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-qt/linguist-tools
+	test? ( dev-python/PyQt6[testlib,${PYTHON_USEDEP}] )
+"
+
+distutils_enable_tests unittest
+
+pkg_setup() {
+	# Needed for lrelease
+	export PATH="$(qt5_get_bindir):${PATH}"
```

Comment:
I know the eclass isn't updated for `qt6` yet, but will this work? Does this do what it's supposed to do? 

That function itself is pretty simple to one-line outside the eclass if you just manually define it.

Reply:
Replacing it by `export PATH="${EPREFIX}/usr/$(get_libdir)/qt6/bin:${PATH}"` doesn't work unfortunately, because it needs the tool `lrelease` for translations from dev-qt/linguist-tools and the latest version in the tree is 5.15.8, so not qt6 compatible yet.
Using the above replacement results in `Failed to build translations: [Errno 2] No such file or directory: 'lrelease'` (no build failure though, it continues and finishes the build).
Interestingly, it seems to work with the qt5 binding, because using `export PATH="$(qt5_get_bindir):${PATH}"` results in no error. I couldn't test yet if the translations actually work even with the qt5 lrelease tool.

Maybe we can leave it as it is now, file a version bump bug for dev-qt/linguist-tools and a bug for app-editors/retext that depends on the version bump bug. What do you think?